scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerStateRecoverHelper.java

```diff
@@ -278,8 +278,8 @@ private boolean runningTaskMustBeResetToPending(InternalTask task, RMProxy rmPro
     }
 
     private void setTaskStatusAndIncrementPendingCounter(TaskStatusCounter counter, InternalTask task) {
-        logger.info("Changing task status to " + TaskStatus.PENDING);
-        task.setStatus(TaskStatus.PENDING);
+        logger.info("Changing task status to " + TaskStatus.SUBMITTED);
```

Comment:
it seems weird to have this message at info level. It seems to be the case originally, but if we have 100K tasks, I can imagine the nicely polluted logs.